README.md

```diff
@@ -923,19 +923,33 @@ await index.update({
 
 ### List records
 
-The `listPaginated` method can be used to list record ids matching a particular id prefix in a paginated format. With clever assignment
-of record ids, this can be used to help model hierarchical relationships between different records such as when there are embeddings for multiple chunks or fragments related to the same document.
+The `listPaginated` method can be used to list record IDs matching a particular ID prefix in a paginated format. With
+[clever assignment
+of record ids](https://docs.pinecone.io/guides/data/manage-rag-documents#use-id-prefixes), this can be used to help model hierarchical relationships between different records such as when there are embeddings for multiple chunks or fragments related to the same document.
+
+Notes:
+
+- When you do not specify a `prefix`, the default prefix is an empty string, which returns all vector IDs
+  in your index
+- There is a hard limit of `300` vector IDs if no `limit` is specified. Consequently, if there are fewer than `300`
```

Comment:
Does it default to 300 or is that the limit? I had thought the default was 100. 🤔

Reply:
In the core DB code, the limit is `300`. Let me test it out personally and get back to you (maybe the API limits it to `100`).

Reply:
Ah fantastic catch @austin-denoble ! It is `100` on the API side now. I will change :)